### PR TITLE
fix: Support Delete NFT

### LINKED ISSUE #1677: Support deletion of NFT Tables
## Summary
Currently deletion script does not delete NFT tables from our database. We should support it.

### PR DESCRIPTION
**Pull request by PeterChen13579**
Fixes #1677

### CODE REVIEW

File: tools/cassandra_delete_range/internal/cass/cass.go [hunk 1]
```diff
@@ -376,7 +343,67 @@ func (c *ClioCass) pruneData(
...
+	if !c.settings.SkipNFTokenTable {
+		marker := util.NewMarker(c.settings.Command)
+		if err := marker.EnterTable("nf_tokens"); err != nil {
+			return err
+		}
+
+		log.Println("Generating delete queries for nft tokens table")
+		rowsCount, deleteCount, errCount = c.prepareAndExecuteDeleteQueries(marker, fromLedgerIdx, toLedgerIdx,
+			"SELECT token_id, sequence FROM nf_tokens WHERE token(token_id) >= ? AND token(token_id) <= ?",
+			"DELETE FROM nf_tokens WHERE token_id = ? AND sequence = ?", deleteMethod{deleteGeneral: maybe.Set(true)}, columnSettings{UseBlob: true, UseSeq: true})
```
Comment (cindyyan317) [inline tools/cassandra_delete_range/internal/cass/cass.go]: nf_tokens table is index table of nftpage object. I think it should be deleted in the same way of objects table?

File: tools/cassandra_delete_range/internal/cass/cass.go [hunk 2]
```diff
@@ -376,7 +343,67 @@ func (c *ClioCass) pruneData(
...
+	if !c.settings.SkipNFTokenURITable {
+		marker := util.NewMarker(c.settings.Command)
+		if err := marker.EnterTable("nf_token_uris"); err != nil {
+			return err
+		}
+
+		log.Println("Generating delete queries for nft token URI table")
+		rowsCount, deleteCount, errCount = c.prepareAndExecuteDeleteQueries(marker, fromLedgerIdx, toLedgerIdx,
+			"SELECT token_id, sequence FROM nf_token_uris WHERE token(token_id) >= ? AND token(token_id) <= ?",
+			"DELETE FROM nf_token_uris WHERE token_id = ? AND sequence = ?", deleteMethod{deleteObject: maybe.Set(true)}, columnSettings{UseBlob: true, UseSeq: false})
```
Comment (cindyyan317) [inline tools/cassandra_delete_range/internal/cass/cass.go]: Why UseSeq is false here ?

File: tools/cassandra_delete_range/internal/cass/cass.go [hunk 3]
```diff
@@ -227,15 +193,15 @@ func (c *ClioCass) pruneData(
...
+		marker := util.NewMarker(c.settings.Command)
 		if err := marker.EnterTable("successor"); err != nil {
 			return err
 		}
 
 		log.Println("Generating delete queries for successor table")
 		rowsCount, deleteCount, errCount = c.prepareAndExecuteDeleteQueries(marker, fromLedgerIdx, toLedgerIdx,
 			"SELECT key, seq FROM successor WHERE token(key) >= ? AND token(key) <= ?",
-			"DELETE FROM successor WHERE key = ? AND seq = ?", deleteMethod{deleteGeneral: maybe.Set(true)}, columnSettings{UseBlob: true, UseSeq: false})
+			"DELETE FROM successor WHERE key = ? AND seq = ?", deleteMethod{deleteGeneral: maybe.Set(true)}, columnSettings{})
```
Comment (cindyyan317) [inline tools/cassandra_delete_range/internal/cass/cass.go:204]: 👍  Good consideration! It was confusing.

File: tools/cassandra_delete_range/internal/cass/cass.go [hunk 4]
```diff
@@ -620,19 +649,20 @@ func (c *ClioCass) prepareAndExecuteDeleteQueries(
...
-								_, numErr := c.performDeleteQueries(&info, session, colSettings)
-								atomic.AddUint64(&totalErrors, numErr)
-								atomic.AddUint64(&totalDeletes, uint64(len(info.Data)))
-								if totalDeletes >= counter {
-									log.Printf("... deleted %d queries ...", counter)
-									counter += 1000
-								}
-								// reset back to the deleted query template after finishing executing delete
-								info = deleteInfo{Query: deleteQueryTemplate}
+						// Checks for delete queries after iterating all pages
```
Comment (cindyyan317) [inline tools/cassandra_delete_range/internal/cass/cass.go]: This comment is no longer true 🤔

Comment (PeterChen13579) [reply tools/cassandra_delete_range/internal/cass/cass.go]: oh yea, you're right 😄

**Review by cindyyan317 — APPROVED** [review]
💯  LGTM

**Pull request merged**